src/utils/event-manager/move-input.js

```diff
@@ -1,5 +1,5 @@
 const MOUSE_EVENTS = ['mousedown', 'mousemove', 'mouseup'];
-const MOVE_EVENT_TYPES = ['mousemove', 'pointermove'];
+const EVENT_TYPE = 'pointermove';
```

Comment:
Is this basically a continuation of the work we did in EventManager to move everything over to only `pointer*` events?